NuRadioReco/modules/channelReadoutWindowCutter.py

```diff
@@ -17,10 +17,10 @@ class channelReadoutWindowCutter:
 
     def __init__(self, log_level=logging.NOTSET):
         logger.setLevel(log_level)
-        self.__sampling_rate_warning_issued = False
         self.begin()
 
     def begin(self):
+        self.__sampling_rate_warning_issued = False
```

Comment:
why

Reply:
We can raise the warning for every channel, for every event, or just once. Currently it's raised just once, though I decided to reset it in `begin` instead of the `__init__`. Doesn't make much of a difference.